apps/wallet/src/ui/app/components/sui-apps/index.tsx

```diff
@@ -2,28 +2,39 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import { useFeature } from '@growthbook/growthbook-react';
-import { ArrowUpRight16 } from '@mysten/icons';
 import cl from 'classnames';
-import { useMemo } from 'react';
+import { useMemo, useState, useEffect } from 'react';
+import { useLocation } from 'react-router-dom';
 
-import { useExplorerLink } from '../../hooks/useExplorerLink';
 import { permissionsSelectors } from '../../redux/slices/permissions';
 import { SuiApp, type DAppEntry } from './SuiApp';
 import { SuiAppEmpty } from './SuiAppEmpty';
-import { Button } from '_app/shared/ButtonUI';
 import { Heading } from '_app/shared/heading';
 import { Text } from '_app/shared/text';
-import { ExplorerLinkType } from '_components/explorer-link/ExplorerLinkType';
 import { useAppSelector } from '_hooks';
 import { FEATURES } from '_src/shared/experimentation/features';
-import { trackEvent } from '_src/shared/plausible';
 import { prepareLinkToCompare } from '_src/shared/utils';
 
 import st from './Playground.module.scss';
 
 function AppsPlayGround() {
-    const ecosystemApps =
+    const location = useLocation();
+    const [tagFilter, setTagFilter] = useState<string | null>(null);
+
+    useEffect(() => {
```

Comment:
I think we could remove the `useEffect` and the state and just have the `tagFilter` created on every render